e2e/components/navigation/navigation.readonly.spec.js

```diff
@@ -40,7 +40,7 @@ describe('when logged in', () => {
     });
   });
 
-  describe('as ONC', () => {
+  describe('as an ONC', () => {
```

Comment:
Suggest

```suggestion
  describe('as an ONC User', () => {
```
or

```suggestion
  describe('as ROLE_ONC', () => {
```

The phrase "an ONC" doesn't make sense, as there's no such thing as a single "ONC"

Same comment with the ONC-STAFF